### DISCUSSION TITLE
How do i prevent from Edge dev tools window to pop up

### DISCUSSION THREAD

**Discussion by Ddady1** [zauberzeug/nicegui#2139] [Q&A]
Hi all,

I use Pycharm as my IDE and every time i run my script, another window of Edge dev tools is poping out.
It never before happened to me. Only when i run NiceGUI code.

BTW, i hope you'll improve the documentation like a lot of people said here before because it seems like an excellent package.
Thanks in advance,

**Comment by rodja** [comment]
> another window of Edge dev tools is poping out

I do not use PyCharm and Edge dev tools. Could you elaborate a bit? Is it a new browser window you see? Have you tried the `show=False` option in `ui.run`.

> i hope you'll improve the documentation like a lot of people said here before

We are constantly working on this. You can contribute by either creating pull-request or commenting on the existing discussion around the topic.

**Reply by Ddady1** [reply]
Hi rodja,
Thanks for your fast reply.
I don't use Edge dev tools and that's the strange thing. And, yes, it's another Edge window that pops out with the title "DevTools - 127.0.0.1:8000/ -[InPrivate]. The funny thing is that my default browser is Chrome.
When i'm running the script, 2 windows open up. One is Chrome with the code and the second one is Edge. ANd as i mentioned before, it only happens when i run NiceGUI code.
The `show=False` didn't help.
You can see it in the attached file.
![contractorweb](https://github.com/zauberzeug/nicegui/assets/98704892/d2b3a9ea-928c-4134-827f-a73fd68bbf5c)
